docs/statistics.md

```diff
@@ -314,266 +420,230 @@ either already acknowledged or dropped by TSBPD as too late to be delivered.
 
```

Comment:
Here is a little confusion in this description. There are two different stateous sequence numbers in the socket that are taken into account when regarding the incoming packet's sequence number:
- the top reception sequence
- the ACK sequence

Packets named "belated" are not late (or, not only) towards the top reception sequence, but they are late towards the ACK sequence. It means that these are packets that come too late to be regarded for anything. Packets that are "late", but not "belated", can still seal holes caused by missing packets, and usually these are retransmitted packets. Note that every belated packet means wasted link capacity.